### PR TITLE
Fix formatting of `glif_psc` and `glif_cond` documentation

### PR DESCRIPTION
**Pull request by nicolossus**
Fixes, in particular, the formatting of mathematical objects and also improves the parameter table.

### CODE REVIEW

File: models/glif_psc.h [hunk 1]
```diff
@@ -68,33 +68,41 @@ GLIF model mechanism setting is based on three parameters
...
-< (V_th + th_spike_add)`.
+parameter setting of ``voltage_reset_fraction`` and ``voltage_reset_add`` may lead
+to the situation that voltage is bigger than threshold after reset. In this case,
+the neuron will continue to spike until the end of the simulation regardless the
+stimulated inputs. We recommend the setting of the parameters of these three models
+to follow the condition of
+
+.. math::
+
+    E_L + \mathrm{voltage_reset_fraction} \cdot \left( V_\mathrm{th} - E_L \right)
```
Comment (jessica-mitchell) [inline models/glif_psc.h:104]:
```suggestion
    E_L + \mathrm{voltage\_reset\_fraction} \cdot \left( V_\mathrm{th} - E_L \right)
```

File: models/glif_psc.h [hunk 2]
```diff
@@ -68,33 +68,41 @@ GLIF model mechanism setting is based on three parameters
...
+    + \mathrm{voltage_reset_add} < V_\mathrm{th} + \mathrm{th_spike_add}
```
Comment (jessica-mitchell) [inline models/glif_psc.h:105]:
```suggestion
    + \mathrm{voltage\_reset\_add} < V_\mathrm{th} + \mathrm{th\_spike\_add}
```
The equation does not render (error: Double subscripts: use braces to clarify). Adding the backslashes seems to do the trick. Can you also modify this equation in `glif_cond`.

**Review by jessica-mitchell — CHANGES_REQUESTED** [review]
thanks @nicolossus just a minor change to render the equations.

**Comment by nicolossus** [comment]
@jessica-mitchell Thanks, the equations should then render properly now.

**Review by jessica-mitchell — APPROVED** [review]
Thanks! lgtm

**Comment by jessica-mitchell** [comment]
Merging with 1 reviewer because doc related

**Pull request merged**